CMakeLists.txt

```diff
@@ -109,11 +114,16 @@ find_package(
 # Subdirectories and main executable
 
 add_subdirectory(docs)
-add_subdirectory(third_party EXCLUDE_FROM_ALL)
+if(NOT USE_EXTERN_QUAZIP)
+  add_subdirectory(third_party EXCLUDE_FROM_ALL)
```

Comment:
-1, because there may be third_party code besides quazip. I'd prefer to handle this in `third_party/CMakeLists.txt`. Though, that file only contains QuaZip related stuff at the moment. :)

Reply:
Yeah, I agree.
I initially had excluded the whole third party folder then came back and changed it because I thought that there might be other folders in there(discovered on catch2). I'll have to look at if there is a `NOT` glob wildcard that could be used.